core/src/main/java/org/bitcoinj/quorums/ChainLocksHandler.java

```diff
@@ -471,13 +474,7 @@ void cleanup() {
 
         lock.lock();
         try {
-            Iterator<Map.Entry<Sha256Hash, Long>> it = seenChainLocks.entrySet().iterator();
-            while(it.hasNext()) {
-                Map.Entry<Sha256Hash, Long> entry = it.next();
-                if (Utils.currentTimeMillis() - entry.getValue() >= CLEANUP_SEEN_TIMEOUT) {
-                    it.remove();
-                }
-            }
+            seenChainLocks.entrySet().removeIf(entry -> Utils.currentTimeMillis() - entry.getValue() >= CLEANUP_SEEN_TIMEOUT);
```

Comment:
This was a suggested change that we can now use in an Android library because of desugaring.